src/tsickle.ts

```diff
@@ -1038,17 +1038,17 @@ class Annotator extends ClosureRewriter {
   }
 
   private emitInterface(iface: ts.InterfaceDeclaration) {
-    if (this.options.untyped) return;
-
     // If this symbol is both a type and a value, we cannot emit both into Closure's
     // single namespace.
     let sym = this.program.getTypeChecker().getSymbolAtLocation(iface.name);
     if (sym.flags & ts.SymbolFlags.Value) return;
 
     let docTags = this.getJSDoc(iface) || [];
     docTags.push({tagName: 'record'});
-    this.maybeAddTemplateClause(docTags, iface);
-    this.maybeAddHeritageClauses(docTags, iface);
+    if (!this.options.untyped) {
+      this.maybeAddTemplateClause(docTags, iface);
```

Comment:
I guess it makes sense for templates to get dropped, because all keys are '?', maybe add a comment because the reason for maybeAddTemplateClause is different than maybeAddHeritageClauses.